UBUNTU.md

```diff
@@ -186,51 +206,102 @@ Don't be in a rush, take time to [read this article](http://sebastien.saunier.me
 understanding of what those keys are used for.
 
 
-## Dotfiles (Standard configuration)
+## GitHub CLI
 
-Hackers love to refine and polish their shell and tools. We'll start with a great default configuration provided by [Le Wagon](http://github.com/lewagon/dotfiles), stored on GitHub. As your configuration is personal, you need your own repository storing it, so you first need to fork it to your GitHub account.
+CLI is the acronym of [Command-line Interface](https://en.wikipedia.org/wiki/Command-line_interface).
 
-:arrow_right: [Click here to **fork**](https://github.com/lewagon/dotfiles/fork) the `lewagon/dotfiles` repository to your account.
+In this section, we will install [GitHub CLI](https://cli.github.com/) to perform useful actions with GitHub data directly from the Terminal.
 
-You should arrive on a page that looks like this. Make sure to **select your GitHub account**.
+It should already be installed on your laptop from the previous commands. First you need to **login**:
 
-![](images/fork.png)
+```bash
+gh auth login -s 'user:email' -w
```

Comment:
@barangerbenjamin does this line work in WSL. Testing with a Ubuntu on DigitalOcean (`ssh`) I get:

```bash
! Failed opening a web browser at https://github.com/login/device
  exec: "xdg-open": executable file not found in $PATH
  Please try entering the URL in your browser manually
```

I fear that WSL students might get that one two 🤔